compose/compose-file/compose-versioning.md

```diff
@@ -0,0 +1,301 @@
+---
+description: Compose file reference
+keywords: fig, composition, compose, versions, upgrading, docker
+title: Compose file versions and upgrading
+---
+
+The Compose file is a [YAML](http://yaml.org/) file defining services,
+networks, and volumes for a Docker application.
+
+There are several versions of the Compose file reference available.
+
+- [Version 3](index.md) (most current, and recommended)
+
+- [Version 2 and Version 2.1](compose-file-v2.md)
+
+- [Version 1](compose-file-v1.md)
```

Comment:
We're a little inconsistent with the number of versions in this section.

Could we standardise on "Version 3.x", "Version 2.x"